### PR TITLE
`should_match_body_to` matcher accepts a regex

### PR DESCRIPTION
**Pull request by lpil**
Hello!

I think it would be nice if this plug matcher could also accept a regex. (I found this handy as it's hard to know where the white-space will be in the body, so often my string literals would not match).

Very new to Elixir, so would love to hear get any feedback here as well.

Cheers,
Louis

### CODE REVIEW

File: lib/shouldi/matchers/plug.ex [hunk 1]
```diff
@@ -68,7 +70,15 @@ defmodule ShouldI.Matchers.Plug do
   """
   defmatcher should_match_body_to(expected) do
     quote do
-      assert var!(context).connection.resp_body =~ ~r"#{unquote(expected)}"
+      plug_should_match_body_to(unquote(expected), var!(context))
```
Comment (ericmj) [inline lib/shouldi/matchers/plug.ex]: `=~` accepts both strings and regexes so I would just remove the regex sigil: `assert var!(context).connection.resp_body =~ unquote(expected)`.

**Comment by lpil** [comment]
Ah! That's that's a much nicer way of doing it. How does this look now?

Cheers,
Louis

**Comment by ericmj** [comment]
Beautiful! Thanks.

**Comment by lpil** [comment]
Thank you. :)

**Pull request merged**